notebooks/00_quick_start/run_notebook_on_azureml.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>can we use run.get_metrics() to retrieve the results?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/681/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
+1 I think this is key

Reply:
Agree, this should work  
```
metrics = run.get_metrics()
print(metrics)
```

Reply:
Yup. This indeed works. I didn't know pm.record() is also logged into metrics, but this is good